flink-table/flink-sql-client/src/main/java/org/apache/flink/table/client/cli/CliStrings.java

```diff
@@ -39,82 +39,171 @@ private CliStrings() {
+                                    "Shows all views in the current catalog and the current database. Syntax: \"SHOW VIEWS;\""))
                     .append(
                             formatCommand(
-                                    "USE",
-                                    "Sets the current default database. Experimental! Syntax: \"USE <name>;\""))
+                                    "SHOW MODULES",
+                                    "Shows all enabled module names with resolution order. Syntax: \"SHOW [FULL] MODULES;\""))
+                    // About "LOAD/UNLOAD MODULE" statements commands.
+                    .append("\n")
                     .append(
                             formatCommand(
                                     "LOAD MODULE",
-                                    "Load a module. Syntax: \"LOAD MODULE <name> [WITH ('<key1>' = "
-                                            + "'<value1>' [, '<key2>' = '<value2>', ...])];\""))
+                                    "Loads a built-in or user-defined module. Syntax: \"LOAD MODULE <module_name> [WITH ('<k1>' = '<v1>', ...)];\""))
                     .append(
                             formatCommand(
                                     "UNLOAD MODULE",
-                                    "Unload a module. Syntax: \"UNLOAD MODULE <name>;\""))
+                                    "Unloads a built-in or user-defined module. Syntax: \"UNLOAD MODULE <module_name>;\""))
+                    // About "INSERT" statements commands.
+                    .append("\n")
                     .append(
                             formatCommand(
-                                    "USE MODULES",
-                                    "Enable loaded modules. Syntax: \"USE MODULES <name1> [, <name2>, ...];\""))
+                                    "INSERT INTO",
+                                    "Inserts the results of a SQL SELECT query into a declared table sink."))
+                    .append(
+                            formatCommand(
+                                    "INSERT OVERWRITE",
+                                    "Inserts the results of a SQL SELECT query into a declared table sink and overwrite existing data."))
+                    // About "SET/RESET" statements commands.
+                    .append("\n")
+                    .append(
+                            formatCommand(
+                                    "SET",
+                                    "Sets a session configuration property. Syntax: \"SET <key>=<value>;\". Use \"SET;\" for listing all properties."))
+                    .append(
+                            formatCommand(
+                                    "RESET",
+                                    "Resets a session configuration property. Syntax: \"RESET <key>;\". Use \"RESET;\" for reset all session properties."))
+                    // About "SELECT" statements commands.
+                    .append("\n")
```

Comment:
3 new commands were added recently to tune the classloader of the jobs when submitted to the Flink cluster, could you please document them as well ?
Something along those lines I guess:

```
	.append(
			formatCommand(
					"ADD JAR",
					"Adds the specified jar file to the submitted jobs' classloader. Syntax: \"ADD JAR '/path/to/some/filename.jar'""))
	.append(
			formatCommand(
					"REMOVE JAR",
					"Removes the specified jar file from the submitted jobs' classloader. Syntax: \"REMOVE JAR '/path/to/some/filename.jar'""))
	.append(
			formatCommand(
					"SHOW JARS",
					"Shows the list of user-specified jar dependencies. This list is impacted by the --jar and --library startup options as well as the ADD/REMOVE JAR commands."))
	.append("\n")
```